client/lib/site/index.js

```diff
@@ -50,7 +50,7 @@ Emitter( Site.prototype );
 Site.prototype.attributes = function( attributes ) {
 	attributes = attributes || {};
 
-	for ( var prop in attributes ) {
+	for ( let prop in attributes ) {
```

Comment:
It doesn't look like any of the changes in this file have are related to the purpose of this PR. Can you separate it out into a separate janitorial PR, so we can focus on only the relevant changes?